deploy/helm/sumologic/values.yaml

```diff
@@ -2350,6 +2354,191 @@ otelagent:
 
 ## Configure otelcol
 otelcol:
+  securityContext:
+    ## ToDo: Verify following comment
+    ## The group ID of all processes in the statefulset containers. By default this needs to be otelcol(999).
+    fsGroup: 999
+
+  ## Add custom labels to all otelcol sts pods(logs, metrics, events)
+  podLabels: {}
+
+  ## Add custom annotations to all otelcol sts pods(logs, metrics, events)
+  podAnnotations: {}
+
+  ## Add custom labels to all otelcol svc (logs, metrics, events)
+  serviceLabels: {}
+
+  ## Add custom labels to all otelcol statefulset PVC (logs, metrics, events)
+  pvcLabels: {}
+
+  statefulset:
+    image:
+      repository: public.ecr.aws/sumologic/sumologic-otel-collector
+      tag: 0.0.18
+      pullPolicy: IfNotPresent
+  logs:
+    enabled: true
+    config:
```

Comment:
It will be good to add here comment explaining that it is config for sumologic-otel-collector and add link to https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Configuration.md

Reply:
Added links to configuration in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1697/commits/ab0758d37657069e239ce6ffbf16c535e8e00405.